TestListener.php

```diff
@@ -1,157 +0,0 @@
-<?php
-
-use Doctrine\Bundle\DoctrineBundle\Command\CreateDatabaseDoctrineCommand;
```

Comment:
Why remove this? This should be a part of a default project?

Reply:
This should only be added to a project if we do integration tests, since that is when you need a local database to be created. We should try to encourage writing unit tests instead.